where in the world/where in the world/MapViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  ViewController.swift
+//  where in the world
+//
+//  Created by Li Zhang on 2020-02-11.
+//  Copyright © 2020 Li Zhang. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+protocol FavToMapDelegate: class{
+    func goToFavPlace(placeName: String)
+}
+
+class MapViewController: UIViewController, FavToMapDelegate, CLLocationManagerDelegate{
+    
+    @IBOutlet weak var mapView: MKMapView!{
+        didSet {mapView.delegate = self}
+    }
+    
+    @IBOutlet weak var starButton: UIButton!
+    @IBOutlet weak var infoPlate: UIView!
+    @IBOutlet weak var placeName: UILabel!
+    @IBOutlet weak var placeDescript: UILabel!
+    @IBOutlet weak var Favorites: UIButton!
+    
+    
```

Comment:
Unnecessary new line here.

In general one new line will suffice to separate out chunks of code functionality.